src/OpenFOAM/meshes/primitiveMesh/PrimitivePatch/PrimitivePatch.H

```diff
@@ -307,7 +307,10 @@ public:
             return points_;
         }
 
-
+        const gpuField<PointType>& getPoints() const
+        {
+            return gpuField<PointType>(points_);
```

Comment:
Ah, Of course. Thanks for pointing out. I guessed I missed the warning since I filtered the `wmake` log using `grep` to catch only the lines containing the word `error`. What is the best way to resolve this? Get the set of points using the existing method `points()` and then create a `pointgpuField` inside the calculate function?